cluster.go

```diff
@@ -61,6 +61,20 @@ func (c *ClusterClient) Close() error {
 	return nil
 }
 
+// PoolLen returns the number of active connections in the pool, for instrumentation
```

Comment:
Are you sure that you don't want to return the sum? If not - please update the comment. Your implementation is *very* unusual and specific...